spec/controllers/api/v1/bounty_claims_controller_spec.rb

```diff
@@ -13,7 +13,8 @@
       access_token: person.create_access_token,
       code_url: "https://github.com/bountysource/frontend/pulls/1",
       description: "Lorem ipsum stack",
-      issue_id: issue.id
+      issue_id: issue.id,
+      amount: 10
     }
```

Comment:
don't need this anymore?

Reply:
It shouldn't though. This part refers to the params passed in when calling the API. if you aren't passing in amount, then it shouldn't appear here. If tests are failing, it should be something else

Reply:
Ah ok!